.doc_gen/metadata/services.yaml

```diff
@@ -409,6 +409,21 @@ eventbridge:
   tags:
     product_categories: {'Application Integration'}
   version: eventbridge-2015-10-07
+firehose:
+  long: "&FHLong;"
+  short: "&FH;"
+  sort: Firehose
+  api_ref: firehose/latest/APIReference/Welcome.html
+  expanded:
+    long: Amazon FireHose
+    short: FireHose
+  blurb:  is an extract, transform, and load (ETL) service that reliably captures, transforms, and delivers streaming data to data lakes, data stores, and analytics services.
```

Comment:
Use either main blurb from docs.aws.amazon.com or (sometimes) the first sentence from the welcome page.
```
makes it easy to collect, process, and analyze data streams in real time. 
```
or (maybe better in this case)
```
is a fully managed service for delivering real-time streaming data to AWS destinations and third-party HTTP endpoints.
```